recipes/openmm-atmmetaforce-plugin/meta.yaml

```
@@ -0,0 +1,82 @@
+{% set name = "openmm-atmmetaforce-plugin" %}
+{% set version = "0.2.4.1" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/Gallicchio-Lab/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: cf7975639c285ed0fab441c2ac36e3cbf6e499256a708240d4acfeb562939bae
+
+build:
+  number: 0
+  skip: true  # [win or osx or ( linux64 and cuda_compiler_version in (undefined, 'None') ) ]
```

```suggestion
  skip: true  # [not linux64]
  skip: true  # [cuda_compiler_version in (undefined, 'None')]
```
This is easier to change later if you want to add more platforms.

---

Thank you @carterbox. This OpenMM plugin is currently used exclusively through its python interface because the C++ API does not implement utility functions indispensable for the calculations.

I have committed the suggested change.